crates/router/src/core/admin.rs

```diff
@@ -577,7 +580,7 @@ impl CreateProfile {
                 format!("{}_{}", business_profile.country, business_profile.business);
 
             let profile_create_request = api_models::admin::ProfileCreate {
-                profile_name: Some(profile_name),
+                profile_name: Some(NameType::from_string(profile_name).unwrap_or_default()),
```

Comment:
We should not be using `unwrap_or_default()` here, we should propagate errors. Please follow the same practice everywhere else in this module.